### PR TITLE
Goblin Loan Mechanic

### PR DESCRIPTION
**Pull request by sumankun**
This is the Goblins Loans system. As far as i checked there are no errors in `error.log`
## Changelog:
- Decision for borrowing/paying back `loan`. 
- `Fortunistic` faith has a head of faith, which funtions as bank and called `Fortunistic Reserve System`. All loans are taken and go back to this title. (discussed with Mariuso). Rulers of this title has titlename `Banker`.
- Due to vanilla character interaction `ask gold from head of faith`, this bank can help for all Fortunistic rulers.
- You return the debt paying 4 times (1/4)+25% on top of it. It asks to return 1/4 of debt every 5 years, the first pay out can take a little longer.
- `FRS holder` has `modifier` +10 gold per month

<details>
<summary>Daelin took debt, first pay back</summary>

image
</details>
<details>
<summary>There are three options</summary>

image
image
image
</details>
<details>
<summary>Son successfully inherited the loan</summary>

image
</details>

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log`

# How to test:
- There is minor decision to take loan from goblins
- Periodically (usually 5 years) They ask to return 1/4 debt +25% on top. Or you can pay back though decision
- If you die, your heir inherit the loan

### CODE REVIEW

**Comment by ElMariuso** [comment]
> You made a new character for the title? Wouldn't it make more sense for the Steamwheedle Trade Prince to be the head of faith?

No?

**Comment by Grekopithikos** [comment]
hmm ok nevermind then

**Review by Gaben1025 — APPROVED** [review]
Amazing work, no errors in log,this is exactly what I had in mind LGTM!

File: localization/english/wc_decisions_l_english.yml [hunk 1]
```diff
@@ -13,3 +13,16 @@
  
  create_gorian_empire_de_jure_changes:1 "Becomes [de_jure|E] [kingdoms|E] of the new Gorian Empire"
  create_gorian_empire:1 "You #P restore#! #V The Gorian Empire#!"
+
+ borrow_loan_goblin_decision:0 "Request Loan from the Goblin Bank"
+ borrow_loan_goblin_decision_tooltip:0 "Request a Loan through the nearest Goblin Bank branch"
+ borrow_loan_goblin_decision_desc:0 "One can only hope that this won't be too complicated of a process, but the Goblins aren't exactly known for making matters too easy for folks who want them to throw money out of their pockets. But, the machinations that have been carefully planned for the future of these lands certainly deserve some external investement. The future... it can only belong to those who are strong enough to stride boldly towards it, and if this isn't a bold stride, what is? The Goblins will likely see this as a new age of some sort of a dubious partnership, and hope that when comes time, we are forced to remember their gestures of goodwill."
```
Comment (arithon) [inline localization/english/wc_decisions_l_english.yml:19]: I'd go here with "and hope that when the time comes", but I think it's ok like this, too.

File: localization/english/wc_decisions_l_english.yml [hunk 2]
```diff
@@ -13,3 +13,16 @@
...
+ borrow_loan_goblin_decision_confirm:0 "It's a simple loan, nothing more"
+ borrow_loan_goblin_decision_effect_tooltip:1 "You borrow [gold|E][gold_i] from the Goblin Bank. You cannot borrow more money again until you have paid back this loan."
+ borrow_from_goblin_bankers_gold_condition:0 "The #L [GetTitleByKey('k_goblin_bankers').GetNameNoTier]#! doesn't have enough gold"
+ borrow_from_goblin_bankers_gold_condition2:0 "Goblins #N don't trust#! you"
+
+ repay_loan_goblin_decision:0 "Pay off Part of the Loan"
+ repay_loan_goblin_decision_tooltip:0 "To have a debt hanging over me is unbecoming of a [ROOT.Char.GetTitleAsNameNoTooltip] – the time has come to repay the money I owe"
```
Comment (arithon) [inline localization/english/wc_decisions_l_english.yml:26]: Shouldn't this end with a . ?

File: localization/english/wc_decisions_l_english.yml [hunk 3]
```diff
@@ -13,3 +13,16 @@
...
+ repay_loan_goblin_decision_desc:0 "You #V will pay#! a #P quarter#! of the initial debt and #N 25%#! on top\n\nIt is high time these lands were eased of their burdens, and what greater burden is there on any land to be financially dependent on external forces that seek to scrounge off it? Indeed the time has come, and the wealth that the Goblins had provided in the past shall now be returned in plenty."
```
Comment (arithon) [inline localization/english/wc_decisions_l_english.yml:27]:
```suggestion
 repay_loan_goblin_decision_desc:0 "You #V will pay#! a #P quarter#! of the initial debt and #N 25%#! on top\n\nIt is high time these lands were eased of their burdens, and what greater burden is there on any land than to be financially dependent on external forces that seek to scrounge off it? Indeed, the time has come, and the wealth that the Goblins have provided in the past shall now be returned in plenty."
```

**Review by arithon — CHANGES_REQUESTED** [review]
just minor things, good in general.

File: localization/english/wc_decisions_l_english.yml (hunk 2 above)
Comment (sumankun) [reply localization/english/wc_decisions_l_english.yml:26]: > Shouldn't this end with a . ?

Tooltips and confirms without dots

**Review by arithon — APPROVED** [review]
LGTM

**Pull request merged**